esp/src/eclwatch/DFUQueryWidget.js

```diff
@@ -372,23 +372,20 @@ define([
                 DropZoneFolders: true
             });
 
+            this.filter.setValue(this.id + "MaxNumberOfFiles", 1000);
+            this.filter.emit("apply");
             this.initWorkunitsGrid();
 
             this.filter.on("clear", function (evt) {
                 context.refreshGrid();
             });
             this.filter.on("apply", function (evt) {
-                context.refreshGrid();
-            });
-            topic.subscribe("hpcc/dfu_wu_completed", function (topic) {
-                context.refreshGrid();
-            });
-
-            WsDfu.DFUQuery({
-                request: {}
-            }).then(function (response) {
-                if (lang.exists("DFUQueryResponse.Warning", response)) {
-                    if (response.DFUQueryResponse.Warning) {
+                var retVal = context.filter.toObject();
+                WsDfu.DFUQuery({
+                    request: {}
+                }).then(function (response) {
+                if (lang.exists("DFUQueryResponse.DFULogicalFiles.DFULogicalFile", response)) {
+                    if (response.DFUQueryResponse.NumFiles > 100000 && !retVal.MaxNumberOfFiles) {
```

Comment:
I don't think so - now that we have a default filter when the user enters the page, they will no longer get the annoying 
"to many files" warning.
If the user then removes that filter they should get the warning as its a real warning.
Also you code prevents all the other possible warning from being emitted.

IOW I think the original test ```lang.exists("DFUQueryResponse.Warning", response) && response.DFUQueryResponse.Warning``` is still correct and you don't need to replace it?